source/clear-linux/get-started/virtual-machine-install/gce.rst

```diff
@@ -1,31 +1,46 @@
 .. _gce:
 
-Launch |CL-ATTR| Compute Engine on Google\* Cloud Platform
+Launch |CL-ATTR| Compute Engine on Google Cloud Platform\*
 ##########################################################
 
 This tutorial walks you through the steps to create a virtual machine
-instance of |CL-ATTR| on `Google Cloud Platform`_ (GCP).
+instance of |CL-ATTR| on `Google Cloud Platform`_ (:abbr:`GCP (Google Cloud Platform)`).
 
-#. Sign in your Google account on the
+.. contents:: :local:
+   :depth: 1
+
+Prerequisites
+*************
+
+** TBD **
```

Comment:
@ttzeng  Can you provide details here?

Reply:
@MCamp859, the only prerequisites I can think of are (1) the reader should already have a Google account, and a billing account set up in GCP in Step 1; and (2) the user SSH key is generated and installed in the Linux PCs that is used to connect to the VMs in GCP (last bullet in Step 13, and Step 15)